pkg/commands/server/pluginshealth.go

```diff
@@ -14,13 +13,21 @@ const (
 	// pluginsHealthCmdUsage is the usage text for the 'plugins health' command.
 	pluginsHealthCmdUsage = "Get a list of plugins' health that are configured with Synse Server"
 
+	// pluginsHealthCmdArgsUsage is the argument usage for the `plugins health` command.
+	pluginsHealthCmdArgsUsage = "[PLUGIN TAG]"
```

Comment:
in my comment above, if we use the variadic function, we could support multiple plugin tags, so I think this would be `[PLUGIN TAG ...]` in that case